emData/download.sh

```diff
@@ -5,8 +5,8 @@
 memprints_url="https://cernbox.cern.ch/index.php/s/CipX7CfTXIj1lcK/download"
 luts_url="https://cernbox.cern.ch/index.php/s/UDSvClVZksBr1Pq/download"
 # Combined modules
-memprints_url_cm="https://www.dropbox.com/s/lf088lvyvg2t6jh/MemPrintsCombined_210319.tgz?dl=0"
-luts_url_cm="https://www.dropbox.com/s/legrvm3gyu5hrth/LUTsCombined_210319.tgz?dl=0"
```

Comment:
@bryates I just realised, we're allowing the MP to fail in the CI anyway, so you could just revert back to the old memprints/luts? Then we don't have to wait for another PR

Reply:
That's a good point. I didn't think we wanted to hold up this PR for the other, anyway. I've just pushed this change, reverting to the master branch test vectors.

Reply:
Actually, this makes the TP quality check fail.

Reply:
The test vectors from March 19th only has `TP_L1L2D_usereg.tab` but the TP looks for `TP_L1L2.tab`. I forgot this was why I switched to the ones I was using. Do we really need to hold up this PR, or can we just use the newer ones I was using? Maybe @aehart and @pwittich can weigh in.

Reply:
I'm confused. The TP worked with that tarball before, so why wouldn't it work now if you undid the changes in the TP?  https://github.com/cms-L1TK/firmware-hls/pull/156/files#diff-c75bfca92df89131b6235026f773668dc5e1588e771b3565ad2d62aeab6a9c6b

I can see the "TP_L1.tab" being downloaded (along with "TP_L1L2D_usereg.tab"), which was the LUT you exchanged for the "TP_L1L2.tab".

Reply:
I tracked it down to the changes I had to make to get the newer test vectors to work. I've restored the TP module as well as the find commands for the TP in download.sh to match the master branch.

Reply:
great :-)